### ISSUE TITLE
Provide an argument to filter out certain HTTP error codes

### ISSUE THREAD

**Issue by spkane** [raviqqe/muffet#291]
Make it possible to ignore a set of HTTP error codes, like `-g 403,999`.

**Comment by spkane** [comment]
there might be another approach to handling/fixing this, but this is primarily being mentioned do to URLs like:

https://www.linkedin.com/in/spkane/ (which always returns an **HTTP 999** to `muffet`, despite being a valid URL) and
https://docs.aws.amazon.com/AmazonS3/latest/userguide/ServerLogs.html (which always returns an **HTTP 403** to `muffet`, despite being a valid URL)

**Comment by spkane** [comment]
Based on #292, being able to set some code to ignore and some to report/warn but not error on at a global level would be useful, in addition to eventually also being able to set this on a per-patter basis.

**Comment by raviqqe** [comment]
I also prefer a proposal in #292 to this because users can't know false positives when websites' behavior changes. One concern of ignored status codes per pattern is their complexity...

**Comment by spkane** [comment]
Agreed #292 basically built on this idea I was trying to implement the scanner GitHub action into one of my website deployment processes.
